docs/weave/basics/02-consensus.md

```diff
@@ -0,0 +1,21 @@
+---
+id: consensus 
+title: Consensus 
+sidebar_label: Consensus 
+---
+
+Consensus is the algorithm by which a set of computers come to agreement on which possible state is correct, and thus guarantee one consistent, global view of the state of the system.
+
+## Eventual finality
+
+All PoW systems use eventual finality, where the resource cost of creating a block is extremely high. After many blocks are gossiped, the longest chain of blocks has the most work invested in it, and thus is the true chain. The "true" head of the chain can switch, in a process called "chain reorganization". But the probability of such a reorganization decreases exponentially the more blocks are built on top of it. Thus, in Bitcoin, the "6 block rule" means that if there are 6 blocks build on top of the block with your transaction, you can be extremely confident that no chain reorganization will ever generate a new true chain that does not include that block. Note this is not a guarantee that it cannot happen, just that the cost of doing so becomes so prohibitively high that is very unlikely to ever happen.
+
+Many early PoS systems, such as BitShares, used voting instead of work to mine blocks, but still used the "longest chain wins" consensus algorithm. However, this has the critical [nothing at stake](https://github.com/ethereum/wiki/wiki/Problems#8-proof-of-stake) problem, since the cost of "mining" blocks on 2, 3, or even 100 alternate chains is quite low.
```

Comment:
```suggestion
Many early PoS systems, such as BitShares, used voting instead of work to mine blocks, but still used the "longest chain wins" consensus algorithm. However, this has the critical [nothing at stake](https://github.com/ethereum/wiki/wiki/Problems#8-proof-of-stake) problem, since the cost of "mining" blocks on two, three, or even 100 alternative chains is quite low.
```